src/components/dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+
+import arrowDownIcon from '../../assets/svg/Icons/arrowDown.svg';
+import arrowUpIcon from '../../assets/svg/Icons/arrowUp.svg';
+
+import './Dropdown.css'
+
+const Dropdown = () => {
+  const [isOpened, setIsOpened] = useState(false);
+
+  const handleClick = () => {
+    setIsOpened(!isOpened);
```

Comment:
이전 상태값을 기반으로 현재 상태값을 변경하려는 경우에는 updater function을 사용해주는게 좋습니다~
```suggestion
    setIsOpened((isOpened) => !isOpened);
```